src/util/prolog/prologSession.ts

```diff
@@ -1,13 +1,37 @@
+import fs from "fs";
+import path from "path";
 import { format_answer, type } from "tau-prolog/modules/core";
 import {PrologResult} from "./prologResult";
 
+// RegExp group one will be the file to be imported.
+const fetchPrologImportsRegExp = /:-( |)use_module\(([a-zA-Z1-9]*)\) *\./g;
+const fetchModuleDeclerationsRegExp = /:-( |)module\([^)]*\) *\./g;
+
+export function importFile(relativePath: string): PrologSession {
```

Comment:
`relativePath` means relative to what? The file from which I'm calling the function, or this file, or the location from which the program is run, or...? That's information for which it'd be worth creating a comment.